### ISSUE TITLE
Q&A shows

### ISSUE THREAD

**Issue by reuven** [devchattv/freelancerstopics#27]
Weren't we going to do a Q&A once per month?  Or did those kinda die off?  I liked them, but I realize that the number of participants was relatively low.

**Comment by luca-ing** [comment]
Maybe need to advertise them more? I never figured out how and when to take part (granted, I didn't look at the time).

**Comment by cmaxw** [comment]
Yes, we should do this. We'll get back on scheduling them. I'll see if I can automate the email notifications.
